doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -1572,6 +1601,140 @@ the limits are exceeded, and an event will be raised.
 `max-write-queue-size` and `max-write-queue-cnt` are as the READ variants,
 but then for WRITEs.
 
+Configure HTTP2
```

Comment:
this is new here, is it intended?

Reply:
🤔  👀  I'll check what happened.

Reply:
I think that when I merged the two sections git interpreted that it was a new addition from the commit.

From what I could see, the settings mentioned here exist in six, though.

Should I add them as a separate commit?

Reply:
I think that makes sense, ya.